libraries/SITL/SITL.h

```diff
@@ -66,6 +67,9 @@ struct sitl_fdm {
         struct vector3f_array points;
         struct float_array ranges;
     } scanner;
+
+    // Rangefinder
```

Comment:
Who would have guessed? :-)

Reply:
My variable naming is just too good ;), fixed